### DISCUSSION TITLE
POS Barcode field always in Auto focus

### DISCUSSION THREAD

**Discussion by mrports** [Blair2004/NexoPOS#883] [Ideas]
will be great if you can make the POS Barcode field always in auto focus even after exiting the POS and coming back. its more smooth and save time especially if the user is using a single window for multitasking between POS and other jobs on the software

**Comment by Blair2004** [comment]
Thanks

**Reply by Blair2004** [reply]
implemented for v4.8.x on https://github.com/Blair2004/NexoPOS-4x/commit/e9f92d16ea9728e6b21d93fb237ccf0ba0c96583
